backend/src/managers/FeedManager.ts

```diff
@@ -257,6 +260,7 @@ export default class FeedManager {
     async convertRawPost(forUserId: number, rawPosts: PostRawWithUserData[], format: ContentFormat): Promise<PostInfo[]> {
         const siteById: Record<number, SiteInfo> = {};
         const posts: PostInfo[] = [];
+        const toUpdateHtmlAndParserVersion: {id: number, html: string}[] = [];
```

Comment:
```suggestion
        const postsToUpdateHtmlAndParserVersion: {id: number, html: string}[] = [];
```

Reply:
and I would also rename the method from `convertRawPost` to `convertRawPosts` since we are touching it

Reply:
done